.circleci/config.yml

```diff
@@ -64,3 +85,28 @@ workflows:
       - test-node-10:
           requires:
             - build-node-10
+
+  build-test-publish:
+    jobs:
+      - build-node-8:
+          filters:
+            <<: *filter_version_tag
+      - build-node-10:
```

Comment:
Why do we need to do a step for both node 8 and 10? Can't we just specify the latest node 10?

Reply:
All of our CircleCI builds for this project build and run tests under both node 8 and node 10 to ensure it stays compatible with both. This means that we don't have to dictate to users that they use node 10 to run this tool. Once node 8 reaches the end-of-life as a maintenance LTS release [at the end of this year](https://nodejs.org/en/about/releases/), we should review this setup.

Reply:
Ohh okay, so ideally should this be done in all of our other components?

Reply:
I think it would be a good idea, yes, but it's worth a wider discussion. The parallel building and testing in node v8 and v10 was something that Sam and I tested and implemented when we were doing some hygiene chores on this repository a couple of weeks ago.

Reply:
Background discussion that inspired this approach: https://github.com/Financial-Times/tooling-helpers/issues/60

Reply:
👍 thanks, I'll read up on that!